Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM docker.io/library/golang:latest AS build-image
+
+ARG BUILD_VERSION="latest"
+
+RUN go install github.com/gammazero/nexus/v3/nexusd@${BUILD_VERSION}
+
+FROM docker.io/library/busybox:latest AS release-image
+
+ARG BUILD_VERSION="latest"
```

Comment:
Do we need the second arg definition? As they are the same. Whenever we need to distinguish them - we can always introduce new arg. What do you think?

Reply:
O thnx for the detailed description, my knowledge in docker/containers seems to be not so deep!